src/webview/mod.rs

```diff
@@ -232,6 +232,15 @@ pub struct WebViewAttributes {
 
   /// Whether all media can be played without user interaction.
   pub autoplay: bool,
+
+  /// Set a handler closer to process chages to navigation.
+  pub on_page_navigating_handler: Option<Box<dyn Fn(String)>>,
```

Comment:
I feel like this clashes with `navigation_handler`, maybe we should change it to `on_page_load_start`

Reply:
I actually started there, but then backed it out because load start seemed misleading because this triggers before even a response has been returned. I agree that the name as is has high potential for confusion and should be changed.

Would on_navigation_start, on_location_change or on_page_change be better names?

Reply:
Maybe `on

Reply:
And then explain that it is fired after `navigation_handler`